course_inventory/inventory.py

```diff
@@ -234,10 +234,21 @@ def pull_sis_section_data_from_udw(section_ids: Sequence[int], conn: connection)
     return udw_section_df
 
 
+def get_course_info_from_db(db_creator_obj: DBCreator) -> pd.DataFrame:
+    logger.info("getting the course info from Database")
+    course_from_db_df = pd.read_sql(f'''select canvas_id, published_at from course 
+                            where workflow_state = 'available' ;''',
+                                    db_creator_obj.engine)
+    logger.info(course_from_db_df.head())
+    return course_from_db_df
+
 # Entry point for run_jobs.py
 
-def run_course_inventory() -> Sequence[DataSourceStatus]:
+
+def run_course_inventory() -> Sequence[Dict[str, Union[ValidDataSourceName, pd.Timestamp]]]:
```

Comment:
Not sure what happened here, but what was there before was correct. Also, I'd like to have two newlines above the comment and underneath the new function you added.